zeek

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+#Sample start/stop script for Zeek running inside docker
+#based on service_script_template v0.2
+#Many thanks to Logan for his Active-Flow init script, from which some of the following was copied.
+#V0.3.2
+
+if [ -n "$1" -a -z "$2" ]; then
+	case "$1" in
+	start|stop|restart|force-restart|status|reload|enable|disable|pull|update)
+		action="$1"
+		;;
+	*)
+		echo "Unrecognized action $1 , exiting" >&2
+		exit 1
+		;;
+	esac
+else
+	echo 'This script expects a single command line option (start, stop, restart, status, reload, enable or disable).  Please run again.  Exiting' >&2
+	exit 1
+fi
+
+host_zeek='/usr/local/zeek'
```

Comment:
Should we default this to `/opt/zeek` since that's what we'll be using?

Reply:
Sounds good, I'll make the change.

Reply:
Did you also want to change /usr/local/zeek/logs/ to /opt/local/zeek/logs/ inside the container?